site2/docs/functions-runtime.md

```diff
@@ -53,6 +53,12 @@ kubernetesContainerFactory:
   jobNamespace:
   # the docker image to run function instance. by default it is `apachepulsar/pulsar`
   pulsarDockerImageName:
+  # the docker image to run java function instance. by default it is `null`
+  javaFunctionDockerImageName:
+  # the docker image to run python function instance. by default it is `null`
+  pythonFunctionDockerImageName:
+  # the docker image to run go function instance. by default it is `null`
+  goFunctionDockerImageName:
```

Comment:
If these three image name is to overwrite `pulsarDockerImageName`, we'd better tell users uses `pulsarDockerImageName` to run function instance if value of `javaFunctionDockerImageName` is absent.

Reply:
@codelipenghui fixed done,  PTAL again, thanks